### DISCUSSION TITLE
Tokens and RREF

### DISCUSSION THREAD

**Discussion by vrcdrafter** [CE-Programming/toolchain#435] [Q&A] [answered]
It seems that the primary use for Tokens in the toolchain is to build Y variables . 

Is there any ways to use Tokens to invoke their actual function . EX run a RREF on a MATRIX A ? 

( again sorry for hitting this discussion so much ) I don't have access to discord.

**Comment by adriweb** [comment, marked as answer]
Yes take a look at the [eval example](https://github.com/CE-Programming/toolchain/tree/master/examples/standalone_examples/eval).

**Reply by vrcdrafter** [reply]
ohh my gosh I totally overlooked this ! Ok thank you , I should be good for a while now . That was the Keystone to what I want to do . 
![image](https://user-images.githubusercontent.com/102326953/234592366-516dad83-f1d7-46c8-9329-db437d9a600d.png)
